From 8e83402db866ae7a67582da28aa68c83545f13c8 Mon Sep 17 00:00:00 2001
From: Alexey Sokolov <sokolov@google.com>
Date: Fri, 26 Aug 2022 01:36:53 +0100
Subject: [PATCH] Don't force Qt definitions onto users

It breaks users who use "signals" keyword and convert const char* to
QString.
---
 QXlsx/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/QXlsx/CMakeLists.txt b/QXlsx/CMakeLists.txt
index aa774b1..a6b1944 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -140,7 +140,7 @@ add_library(QXlsx
 
 add_library(QXlsx::QXlsx ALIAS QXlsx)
 
-target_compile_definitions(QXlsx PUBLIC
+target_compile_definitions(QXlsx PRIVATE
     -DQT_NO_KEYWORDS
     -DQT_NO_CAST_TO_ASCII
     -DQT_NO_CAST_FROM_ASCII
